helm/grafana-app/values.yaml

```diff
@@ -620,19 +631,21 @@ imageRenderer:
   enabled: false
   replicas: 1
   image:
-    registry: quay.io
     # image-renderer Image repository
-    repository: giantswarm/grafana-image-renderer
+    repository: grafana/grafana-image-renderer
     # image-renderer Image tag
-    tag: 2.0.0
+    tag: latest
```

Comment:
is this an upstream change?

Reply:
Yes, right, should use our registry, will change that back.